.buildkite/custom-tests.json

```diff
@@ -48,6 +48,12 @@
       "command": "cd fuzz && cargo test --package common --lib blk && cargo +nightly fuzz run blk -- -max_total_time=900 -timeout=60s",
       "platform": ["x86_64", "aarch64"],
       "timeout_in_minutes": 20
+    },
+    {
+      "test_name": "prove-virtio-queue",
+      "command": "cargo kani --package virtio-queue",
+      "platform": ["x86_64", "aarch64"],
+      "timeout_in_minutes": 20
```

Comment:
nit: currently it finishes in just 2 minutes, so maybe we can leave the default timeout for now?

Reply:
It makes more sense to only have override the timeout if it's needed, I agree with @roypat